conf/broker.conf

```diff
@@ -1264,10 +1264,10 @@ replicatorPrefix=pulsar.repl
 # due to missing ZooKeeper watch (disable with value 0)
 replicationPolicyCheckDurationSeconds=600
 
-# Default message retention time
+# Default message retention time. Using a value of -1, is disabling message retention time limit
 defaultRetentionTimeInMinutes=0
 
-# Default retention size
+# Default retention size. Using a value of -1, is disabling message retention size limit
```

Comment:
I think it's not correct.

0 means the retention is disabled,
-1 means the data will not be removed by time or size quota?

I think we can change to

Reply:
Great

Reply:
Great.
I find this PR https://github.com/apache/pulsar/pull/16572 has already fix.  But the description in the doc maybe not clear enought, e. g, :
- https://github.com/apache/pulsar/blob/f02679d83402ac07e06c48570505f343e350a491/pulsar-broker-common/src/main/java/org/apache/pulsar/broker/ServiceConfiguration.java#L2382
`will not be removed by time or size quota` is better I think

- https://github.com/apache/pulsar/blob/f02679d83402ac07e06c48570505f343e350a491/pulsar-broker-common/src/main/java/org/apache/pulsar/broker/ServiceConfiguration.java#L2388
`no infinite size quota` should be `infinite size quota`

So I rewrite it as you mentioned above.  @codelipenghui .  PTAL also @Anonymitaet ~ thanks